bundle/Makefile

```diff
@@ -27,9 +37,18 @@ install:
 	install $(SELINUX) -D -m 644 -t $(BASHINSTALLDIR) completions/bash/crio
 	install $(SELINUX) -D -m 644 -t $(FISHINSTALLDIR) completions/fish/crio.fish
 	install $(SELINUX) -D -m 644 -t $(ZSHINSTALLDIR) completions/zsh/_crio
+	install $(SELINUX) -D -m 644 -t $(CNIDIR) contrib/10-crio-bridge.conf
+	install $(SELINUX) -D -m 644 -t $(CONTAINERS_DIR) contrib/policy.json
+	install $(SELINUX) -D -m 644 -t $(SYSTEMDDIR) contrib/crio-shutdown.service
+	install $(SELINUX) -D -m 644 -t $(SYSTEMDDIR) contrib/crio-wipe.service
+	install $(SELINUX) -D -m 644 -t $(SYSTEMDDIR) contrib/crio.service
+	install $(SELINUX) -D -m 644 -t $(CRIOWIPEDIR) contrib/crio-wipe.bash
+	install $(SELINUX) -D -m 644 -t $(CRIOWIPEDIR) contrib/helpers.bash
+	install $(SELINUX) -D -m 644 -t $(CRIOWIPEDIR) contrib/lib.bash
```

Comment:
and these (but we still need crio-wipe.service above)

Reply:
Yes, I removed everything except the service